src/components/meta-data/index.jsx

```diff
@@ -0,0 +1,496 @@
+					metaData.keywords = [];
+					gc.taxonomy.tags.forEach((item) => {
+						if (item.slug) metaData.keywords.push(item.slug);
+					});
+					metaData.keywords = metaData.keywords.join(",");
+				}
+			} else {
+				metaData.keywords = null;
+			}
+
+			storyMetaDataTags = (
+				<>
+					{metaData.description && (
+						<>
+							<meta name="description" content={metaData.description} />
+							<meta name="twitter:description" content={metaData.description} />
+							<meta property="og:description" content={metaData.description} />
+						</>
+					)}
+					{metaData.keywords && <meta name="keywords" content={metaData.keywords} />}
+
+					<meta property="og:title" content={metaData["og:title"]} />
+					<meta name="twitter:title" content={metaData["twitter:title"]} />
+
+					{metaData.ogImage && <meta property="og:image" content={metaData.ogImage} />}
+					{metaData.ogImageAlt && <meta property="og:image:alt" content={metaData.ogImageAlt} />}
+					{metaData.twitterImage && <meta name="twitter:image" content={metaData.twitterImage} />}
+					{metaData.twitterImageAlt && (
+						<meta name="twitter:image:alt" content={metaData.twitterImageAlt} />
+					)}
+
+					{pageType === "article" && (
+						<>
+							<meta property="og:type" content="article" />
+							<meta name="robots" content="noarchive" />
+						</>
+					)}
+				</>
+			);
+		}
+	} else if (pageType === "author") {
+		const author = gc && gc.authors && gc.authors.length ? gc.authors[0] : {};
+		const fallbackTitle = (author.byline && `${author.byline} - ${websiteName}`) || websiteName;
+		metaData.description = metaValue("description") || author.bio || null;
+		metaData["og:title"] = metaValue("og:title") || fallbackTitle;
+		metaData["twitter:title"] = metaValue("twitter:title") || fallbackTitle;
+		metaData.title = metaValue("title") || fallbackTitle;
+		const { name: authorName } = author;
+		const authorImageUrl =
+			typeof author.image === "string" ? author.image : author.image && author.image.url;
+		const authorAltText =
+			typeof author.image === "object" ? author.image.alt_text : author.byline || authorName;
+
+		const authorPhoto = authorImageUrl || metaData.fallbackImage;
+		const authorAlt = authorAltText || authorName || author.byline || websiteName;
+
+		authorMetaDataTags = (
+			<>
+				{metaData.description && (
+					<>
+						<meta name="description" content={metaData.description} />
+						<meta property="og:description" content={metaData.description} />
+						<meta name="twitter:description" content={metaData.description} />
+					</>
+				)}
+				<meta property="og:title" content={metaData["og:title"]} />
+				<meta name="twitter:title" content={metaData["twitter:title"]} />
+				{authorPhoto && (
+					<>
+						<meta property="og:image" content={authorPhoto} />
+						<meta property="og:image:alt" content={authorAlt} />
+						<meta name="twitter:image" content={authorPhoto} />
+						<meta name="twitter:image:alt" content={authorAlt} />
```

Comment:
The author images aren't being resized and should be.

Reply:
I'm guessing the author images weren't previously being resized because they're all square images, so the default aspect ratio of 1200x630 didn't work. I'm pushing a PR to resize the author photos to 1200x1200 (which seems to be the recommended size for square images: https://whatabout.dev/formats-for-open-graph-images)